### ISSUE TITLE
Taskbar rendering issues on second monitor

### ISSUE THREAD

**Issue by Queex** [Open-Shell/Open-Shell-Menu#2067]
### Describe the bug

When the taskbar hides and unhides on the second monitor, the custom start menu icon is misaligned.

![Clipboard02](https://github.com/user-attachments/assets/58ef3445-7419-4b39-8e23-70cebc06f3d9)

This does not occur on the primary monitor. I have not tested if it also happens with the default icon.

There is also a flickering white line at the bottom of the area of the taskbar icons when the bar hides and unhides.

### Area of issue

Taskbar

### To reproduce

- Have task bat set to autohide, and the task bar set to show on all monitors.
- Hover the mouse over the bottom of the screen to make the taskbar unhide.
- Move the mouse away to make the taskbar hide.

### Expected behavior

The rendering to remain consistent with previous version of OpenShell, and with the primary monitor.

### Open-Shell version

4.4.194

### Windows version

Windows 10 22H2

**Comment by ge0rdi** [comment]
> The rendering to remain consistent with previous version of OpenShell

So it worked correctly in previous `Open-Shell` versions, right?

**Comment by Queex** [comment]
Yes, neither the alignment issue nor the flickering white line appeared before I updated it most recently.

**Comment by ge0rdi** [comment]
Hmm, I'm unable to reproduce such behavior :(

Could you post better screenshots (that show whole taskbar, not clipped)?
Ideally one from good state and one from bad state.

Do you use some other customization software?
Do you have touch screen (and thus tablet optimized mode)?

**Comment by ge0rdi** [comment]
> There is also a flickering white line at the bottom of the area of the taskbar icons when the bar hides and unhides.

I can see this with `4.4.191` too, when I have some white window behind the taskbar.
Though it happens only for secondary taskbar. Probably we (or Windows?) handle that one a bit differently.

So this issue doesn't seem like regression. It was there before.

**Comment by ge0rdi** [comment]
> I can see this with 4.4.191 too, when I have some white window behind the taskbar.

I fact, I see such behavior even with original Windows taskbar (`Open-Shell` not loaded).

**Comment by Ecks32** [comment]
I'm having the same issue with the custom image on the second monitor.  The image doesn't move at the same speed as the taskbar and occasionally doesn't completely hide, about half sticks out.   I never noticed this behavior on previous releases. 

I have my taskbar on the left side of my monitors and keep it hidden when not in use.  It doesn't cause any problems with the taskbar or open windows.  If I catch it again I'll try to get a screen shot.  If I can figure out how I'll try to capture the movement issue.

**Comment by Queex** [comment]
Grabbed some action shots of the whole desktop:

Task bar showing:
![Clipboard01](https://github.com/user-attachments/assets/e40c6ab8-5d7e-4379-b753-7e78f28a496b)

Taskbar unhiding:
![Clipboard02](https://github.com/user-attachments/assets/57e79f29-7b92-41a2-94a9-08c2d7526a1a)

Taskbar hiding:
![Clipboard03](https://github.com/user-attachments/assets/388fbc79-b907-4402-9e3e-f4c6bc9e0995)

Windows taskbar settings, in case it matters:
Use small taskbar buttons OFF
Combine taskbar buttons ALWAYS, HIDE LABELS
-
Show on all displays ON
Show taskbar buttons on ALL TASKBARS
Combine buttons on other taskbars ALWAYS, HIDE LABELS

Although an interesting finding in the OpenShell settings. The custom size was set to 40, but the image used for the custom button is 80x240. This would explain the discrepancy with the hide/unhide animation, but if I set that size to 80 or 0 (to use the original size) the button displays twice as large as it should on both taskbars:
![Clipboard01](https://github.com/user-attachments/assets/0f96999f-d8fb-4609-be53-5d69c8c16472)

Here's the image I'm using for the custom icon:
![2k_start](https://github.com/user-attachments/assets/f2fce082-78f9-4a01-9be2-f72a612d7f08)

**Comment by Ecks32** [comment]
Here is a screen shot of the button sticking out when the taskbar is hidden, and a video of the monitor showing how the buttons moves in relation to the taskbar.  The black on the left and top is the bezel.  This seems to be just cosmetic, I haven't noticed it causing any problems. 

![mon2_taskbar](https://github.com/user-attachments/assets/13cf3227-acf6-4c9b-afba-f9406473e965)

https://github.com/user-attachments/assets/36f4dc8a-e618-4a0b-8fba-055d3beee2fb

**Comment by ge0rdi** [comment]
Guys, I finally get what you mean and see what the issue is (change in code due to touch-scree devices support).
Working on fix.

**Comment by ge0rdi** [comment]
Fixed in `4.4.195`.
